R-package/R/xgb.DMatrix.R

```diff
@@ -17,13 +20,13 @@
 #' xgb.DMatrix.save(dtrain, 'xgb.DMatrix.data')
 #' dtrain <- xgb.DMatrix('xgb.DMatrix.data')
 #' @export
-xgb.DMatrix <- function(data, info = list(), missing = NA, ...) {
+xgb.DMatrix <- function(data, info = list(), missing = NA, silent = FALSE, ...) {
```

Comment:
I've already done that [in this commit](https://github.com/dmlc/xgboost/pull/2237/commits/d9ccf822ba70c3943100989945f6d1ad54453e36#diff-bea79b5e41f6840bd1ea0f096cbba19c) (`regexp=NA` means expect no output).